data/transport missions.txt

```diff
@@ -1975,3 +1975,45 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "Transfer enforcers to <planet>"
+	description "Transport 15 enforcers from <source> to <destination> by <day> to fulfill a request from the New Boston loan broker."
+	deadline
+	passengers 15
+	minor
+	to offer
+		random < 25
+		"credit score" > 750
+		"net worth" > 5000000
+	source "New Boston"
+	destination
+		government "Syndicate"
+	on offer
+		conversation
+			`As you're walking through the spaceport searching for a bar or pub, you notice one person who stands out. A nearly bald, middle-aged man wanders through the crowd in an expensively tailored suit, searching the faces that pass by him. You pause and look at him more closely; he seems oddly familar. Eventually, his gaze settles on you, making eye contact. He stares, then begins moving through the crowd toward you.`
+			choice
+				`	(Wait for him to reach you.)`
+				`	(Walk forward to meet him.)`
+				`	(Turn the other way and lose the man in the crowd.)`
+					decline
+			`	When you and the man come face to face, he studies you for a moment. You realize who he is when you hear his voice: "Captain <last>, correct? I suppose I should formally introduce myself, even though we've met before. I'm Mr. Johnson," he says, extending a hand. You shake it, wondering how the loan broker who lent you money to purchase your first ship could possibly recognize you. "Before you ask, I pay people to keep tabs on all my clients, new and old. I find that it can pay off to have contacts from several different walks of life. When I was informed a ship registered in your name was in the New Boston spaceport, I realized that this could be an opportunity for both of us, and decided to come find you. It was a bit hard to recognize you, since the only picture I have isn't exactly recent, but it seems it wasn't that big of a problem," he explains.`
+			`	 "I've found myself in a predicament, and you can assist in solving it. One of my... 'clients' hasn't been very proactive in paying their dues on time. This wouldn't be a problem for any normal client, but this man happens to be in charge of a small street gang on <destination>. Of course, there's a simple fix: send a group of paid enforcers to convince the client to be more... accountable, let's say."`
+			choice
+				`	"So what do I have to do with this?"`
+				`	"Sorry, but I'm too busy to start another job right now."
+					decline
+			`	"Usually I have a different contact that I could use for this kind of job, but she seems to be unavailable at the moment. So, I need a ship to carry my fifteen men to <destination>. I've been told you're the kind of person who might be interested in such a job," Mr. Johnson concludes, looking at you expectantly.`
```

Comment:
Couldn't he hire any old ship to bring his goons to the destination system? It's specifically mentioned that you're not doing anything illegal or dangerous. Why would he specifically want or need you, then?